Eslint Lints `README.md` File (And Fails)

https://github.com/kossiitkgp/YAKW/blob/79d43afb2dade94e69a285c26de66a7a5a197c2f/package.json#L28-L33

Due to this line, all staged `.md` files are checked, and ESLint gives the following error on the `README.md` file.
```
✖ eslint --ext .js,.astro --fix:

[...]/YAKW/README.md
  1:1  error  Parsing error: Unexpected character '#'
```

--- a/astro.config.mjs
+++ b/astro.config.mjs
@@ -5,5 +5,5 @@ import vercel from "@astrojs/vercel/serverless";
 // https://astro.build/config
 export default defineConfig({
   output: "server",
-  adapter: vercel()
+  adapter: vercel(),
 });

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "typescript": "5.0.3"
   },
   "lint-staged": {
-    "*.{astro,md}": [
+    "*.{astro,js}": [
       "prettier --write",
       "eslint --ext .js,.astro --fix"
     ]

--- a/package.json
+++ b/package.json
@@ -26,9 +26,11 @@
     "typescript": "5.0.3"
   },
   "lint-staged": {
-    "*.{astro,js}": [
-      "prettier --write",
+    "*.{astro,js,ts}": [
       "eslint --ext .js,.astro --fix"
+    ],
+    "*": [
+      "prettier --write"
     ]
   }
 }

